faq-bundle/src/Resources/contao/models/FaqModel.php

```diff
@@ -69,6 +75,9 @@
  * @method static Collection|FaqModel[]|FaqModel|null findByAlias($val, $opt=array())
  * @method static Collection|FaqModel[]|FaqModel|null findByAuthor($val, $opt=array())
  * @method static Collection|FaqModel[]|FaqModel|null findByAnswer($val, $opt=array())
+ * @method static Collection|FaqModel[]|FaqModel|null findByPageTitle($val, $opt=array())
+ * @method static Collection|FaqModel[]|FaqModel|null findByRobots($val, $opt=array())
+ * @method static Collection|FaqModel[]|FaqModel|null findByDescription($val, $opt=array())
```

Comment:
Please also add the `countBy*()` methods.

Reply:
Added with 5872196.